src/libraries/sendtohelixhelp.proj

```diff
@@ -362,7 +375,7 @@
       <HelixWorkItem Include="@(_RunOnlyWorkItem -> '%(FileName)')" >
         <PayloadArchive>%(Identity)</PayloadArchive>
         <!-- No RunTests script generated for the sample project so we just use the direct command -->
-        <Command>$(ExecXHarnessCmd) wasm $(XHarnessCommand)  --app=. --browser=Chrome  --html-file=index.html --output-directory=$(XHarnessOutput) -- %(FileName).dll --testing</Command>
+        <Command>$(ExecXHarnessCmd) wasm $(XHarnessCommand)  --app=. --browser=Chrome $(XHarnessBrowserPath) --html-file=index.html --output-directory=$(XHarnessOutput) -- %(FileName).dll --testing</Command>
```

Comment:
It seems that linux would find it on PATH, why windows are different ?

Reply:
Probably some difference in xharness. At least it has the option this time. For JS engines I had to modify xharness to look for them.